hoomd/md/pair/aniso.py

```diff
@@ -256,21 +256,48 @@ class ALJ(AnisotropicPair):
     .. _Ramasubramani, V.  et. al. 2020: https://doi.org/10.1063/5.0019735
 
     .. math::
-        U(r, r_c) = 4 \varepsilon \left[ \left( \frac{\sigma}{r} \right)^{12} -
-            \left( \frac{\sigma}{r} \right)^{6} \right] +
-            4 \varepsilon_c \left[ \left( \frac{\sigma_c}{r_c} \right)^{12} -
-            \left( \frac{\sigma_c}{r_c} \right)^{6} \right]
-
-    The first term is the standard center-center interaction between two
-    Lennard-Jones spheres. The second term is a contact interaction computed
-    based on the smallest distance between the surfaces of the two shapes,
-    :math:`r_c`. The total potential energy can thus be viewed as the sum of
-    two interactions, a central Lennard-Jones potential and a shifted
-    Lennard-Jones potential where the shift is anisotropic and depends on the
-    extent of the shape in each direction.
+
```

Comment:
This comment applies a few lines above:

```
    `ALJ` computes the Lennard-Jones force between anisotropic particles as
    described in `Ramasubramani, V.  et. al. 2020`_, using the formula:
```
Remove the "we" as the documentation is not written from the point of view of the authors.